### PR TITLE
Fix event handling in the RDM test server.

### LINKED ISSUE #948: Improve RDM Test Server event handling
@nomis52 thinks we should create our own Event which holds the result args and a threading.Event object much like the Future does in OLA. He's worried about the global since his understanding is that it really is global.

https://github.com/OpenLightingProject/ola/blob/master/tools/rdm/rdm_test_server.py#L112

### PR DESCRIPTION
**Pull request by nomis52**
Closes #948

### CODE REVIEW

**Comment by peternewman** [comment]
LGTM

File: tools/rdm/rdm_test_server.py [hunk 1]
```diff
@@ -63,6 +63,22 @@ class ServerException(Error):
   """Indicates a problem handling the request."""
 
 
+class OLAFuture(object):
+  def __init__(self):
+    self._event = Event()
+    self._data = None
+
+  def set(self, data):
+    self._data = data;
```
Comment (peternewman) [inline tools/rdm/rdm_test_server.py]: Remove the semicolon.

File: tools/rdm/rdm_test_server.py [hunk 2]
```diff
@@ -108,22 +124,17 @@ def MakeSyncClientCall(self, method, *method_args):
...
-    # TODO(simon): Create our own Event which holds the result args and a
-    # threading.Event object #948
-    event = Event()
+    future = OLAFuture()
 
     def Callback(*args, **kwargs):
-      global args_result
-      args_result = args
-      event.set()
+      future.set(args);
```
Comment (peternewman) [inline tools/rdm/rdm_test_server.py]: Semicolon

**Comment by nomis52** [comment]
Done

**Comment by peternewman** [comment]
LGTM

**Pull request merged**